tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -34,7 +34,7 @@ function SplashPage() {
           <div className="usa-hero__callout">
             <h1 className="usa-hero__heading">
               <span className="usa-hero__heading--alt font-serif-2xl margin-bottom-5">
-                Sign into TANF Data Portal
+                Sign into TANF Data Portal!
```

Comment:
Curious why this change is happening here?

Reply:
So I could look at the front page and see a change from circle ci deploying successfully

Reply:
I see, that makes sense. I presume we will be removing this before putting it into final review?